### PR TITLE
clients: Consistent headers + utility script #534

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

**Comment by vingar** [comment]
problem with pylint on travis failling on
```
PYLINT FAILED
************* Module doc/source/conf.py
F:  1, 0: No module named doc/source/conf.py (fatal)
```

while it should not -- I've removed conf.py from the PR

File: tools/pip-requires [hunk 1]
```diff
@@ -47,3 +47,7 @@ retrying==1.3.3             # general-purpose retrying library to simplify the t
 functools32==3.2.3.post2    # explicitly needed on CC7
 redis==2.10.6               # Python client for Redis key-value store
 numpy==1.13.3               # Numpy for forecasting T3C
+kerberos>=1.2.5             # Kerberos high-level interface
```
Comment (bari12) [inline tools/pip-requires:50]: Something is wrong with this PR. It adds the dependencies we excluded again.

**Review by bari12 — CHANGES_REQUESTED** [review]

**Comment by vingar** [comment]
ah, I wrongly rebased against next to solve a conflict (but this impacted the one on master). I'll split it up  in several PRs.

**Comment by bari12** [comment]
Ok, I'll close this then.

**Pull request closed without merging**